release_notes/v2_vs_v3.adoc

```diff
@@ -16,6 +16,10 @@ Many of the same terms are used, and the same functions are performed, but the t
 
 This topic discusses these differences in detail, in an effort to help OpenShift users in the transition from OpenShift v2 to OpenShift v3.
 
+ifdef::openshift-dedicated[]
+OpenShift Dedicated uses the same code base as OpenShift Enterprise 3. OpenShift Dedicated is hosted on the public cloud and managed by Red Hat. An OpenShift Enterprise 3 subscription entitles you to host and manage the software on your infrastructure of choice.
+endif::[]
```

Comment:
Maybe this can be a Note box. Something like:

> "OpenShift v2" and "OpenShift v3" refer to the second and third architectural revisions of OpenShift, respectively. OpenShift Dedicated 3, referring to the product's major version, uses the same code base as OpenShift Enterprise 3, both based on OpenShift v3.

Gives some context to why this topic is appearing in the OSD docs, and also explains what "v2" and "v3" are referring to. I don't believe that's explained anywhere in the OSE 3.1 docs anymore, since we axed the "What's New" category post-3.0 (there was a similar Note at the end of https://docs.openshift.org/latest/whats_new/index.html, which persists in the Origin docs).

If you're into something like that, I could possibly see it moved up to become the first paragraph instead of a Note box, too.